lib/phoenix/endpoint.ex

```diff
@@ -193,60 +193,47 @@ defmodule Phoenix.Endpoint do
 
   ## Endpoint API
 
-  In the previous section, we have used the `config/2` function that is
+  In the previous section, we have used the `c:config/2` function that is
   automatically generated in your endpoint. Here is a summary of all the
   functions that are automatically defined in your endpoint.
 
   #### Paths and URLs
 
-    * `struct_url()` - generates the endpoint base URL, but as a `URI` struct
-    * `url()` - generates the endpoint base URL without any path information
-    * `path(path)` - generates the path information when routing to this endpoint
-    * `static_url()` - generates the static URL without any path information
-    * `static_path(path)` - generates a route to a static file in `priv/static`
+    * `c:struct_url/0` and `c:url/0` - that generate the endpoint base URL
```

Comment:
Why merge these two? And in any case, there's a superfluous "that" after the `-` :)